badges/badges.md

```diff
@@ -1,26 +1,25 @@
-# Badges!
+# Badges
 
-### What are badges?
-The badges are a complement system to the Ladder - while ladder outlines career paths, the badges reward behaviors and achievements we respect and appreciate in our fellows.
+### What are the badges?
+The badges are a complementing tool to the Ladder - while ladder outlines career paths, badges reward behaviors and achievements we respect and appreciate in our team members.
 
 ### How can I create a badge?
-Simple. Badges must meet the following criteria:
-#### 1. A badge achievement must be a measurable, objective, pass fail criteria.
-###### Example Badge: *Good presenter*
-Do: Presented to an audience of at least 50 people
-Don't: Had a nice, well structured presentation
+Simply. Badges must meet the following criteria:
+#### 1. Badge achievement must be a measurable, objective, and pass-fail criteria.
+###### Example: *Good speaker*
+Surely: Prepared an educative talk for an audience of ~50+ people
+Not really: Uploaded a few well-structured presentations
 
-#### 2. A badge must be achievable by one person in reasonable time
-###### Example Badge: *CR Knight*
-Do: Performed 1000 code reviews
-Don't: Invested 100000 hours reviewing code
+#### 2. A badge must be achievable by one person in a reasonable time
+###### Example: *Code Review Knight*
+Surely: Had objective insights that helped many people get on the right track
+Not really: Invested 1000 hours in reviewing code
 
-#### 3. Badges must reflect behavior that are contributing to the company as well as to the team of the badge winner
-###### Example badge: *CR Knight*
-Do: Performed 1000 code reviews
-Don't: Performed a code review in full plate armor [^1]
+#### 3. Badges must reflect behavior that are contributing to the team, the community and the company
+###### Example: *Mentor Mage*
+Surely: Helped his team-mates understand the codebase so that they became more independent at work
+Not really: Left several instructions for his team-mates of how to proceed
 
 
 ### What are the badges for?
-The Ladder exists to make career progression at Socialbakers engineering department more transparent and easier to understand, and it rewards technical excellence alongside effective behaviors and communication skills.
-There are things that we sometimes do, however, that are great and should be celebrated that do not immediately contribute to promotion chances - being an exceptional code reviewer,
+The Ladder rewards technical excellence alongside effective behaviors and communication skills. It doesn't, however, cover all aspects and abilities one may possess and we should appreciate. Thus, the badges should serve as an additional tool to celebrate accomplishments, despite the fact they don't necessarily lead into a promotion.
```

Comment:
For some reason, it only committed half or my summary, so good catch!   
Your formulation makes more sense.  
The ladder, however, isn't a rewarding mechanism, it's an outline of where you are and where you should be going. We need to clarify at every opportunity we have that the Ladder is a guiding system and nothing more

Reply:
So what do you suggest? Should we add another explanation in this instance or change the current one?

Reply:
I'm pushing the full version again, you can edit it afterwards